jib-maven-plugin/src/test/java/com/google/cloud/tools/jib/maven/BuildDockerMojoIntegrationTest.java

```diff
@@ -110,4 +116,24 @@ public void testExecute_defaultTarget()
             defaultTargetTestProject.getProjectRoot(),
             "default-target-name:default-target-version"));
   }
+
+  @Test
+  public void testExecute_skipJibGoal() throws VerificationException, IOException {
+    String targetImage = "emptyimage:maven" + System.nanoTime();
+
+    Verifier verifier = new Verifier(skippedTestProject.getProjectRoot().toString());
+    verifier.setSystemProperty("_TARGET_IMAGE", targetImage);
+    verifier.setAutoclean(false);
+    verifier.setSystemProperty("jib.skip", "true");
+
+    verifier.executeGoal("jib:" + BuildDockerMojo.GOAL_NAME);
+    File logFile = new File(verifier.getBasedir(), verifier.getLogFileName());
```

Comment:
nit: use `Path`? And `new String(Files.readAllBytes(logFile), StandardCharsets.UTF_8)`

Reply:
Yeah, that way, it will remove the Guava dependency.